services/slam/builtin/orbslam_int_test.go

```diff
@@ -20,6 +20,10 @@ import (
 	"go.viam.com/utils/artifact"
 )
 
+const (
+	dataInsertionMaxTimeoutMin = 3
```

Comment:
[opt] can we change this to sec for more fine tuned control?

Reply:
We can do so in the future if it is necessary to lower the value below a minute. For now, 3 minutes is a good value to fix the issue.